services/web/pkg/config/defaults/defaultconfig.go

```diff
@@ -49,7 +49,8 @@ func DefaultConfig() *config.Config {
 				},
 				Apps: []string{"files", "search", "preview", "text-editor", "pdf-viewer", "external", "user-management"},
 				Options: map[string]interface{}{
-					"hideSearchBar": false,
+					"hideSearchBar":        false,
+					"previewFileMimeTypes": []string{"image/gif", "image/png", "image/jpeg", "text/plain"},
```

Comment:
```suggestion
					"previewFileMimeTypes": []string{"image/gif", "image/png", "image/jpeg", "text/plain"},
```
Didn't @fschade build `tiff` support recently?

Reply:
With latest master, I tried to receive previews for image/tif and image/tiff, but requests give me a 404, so I guess it's not supported yet (on latest master)

Reply:
Then it probably was in experimental. Nevermind, thanks for checking!

Reply:
cc @fschade maybe you need to adapt this for the experimental branch